package.json

```diff
@@ -8,14 +8,43 @@
     "name": "app",
     "main": "src/app.js",
     "dependencies": {
-      "bootstrap": "npm:bootstrap@4.0.0-alpha.2",
+      "bootstrap": "github:twbs/bootstrap@4.0.0-alpha.2",
```

Comment:
This was the root of the problem was that `jspm i github:twbs/bootstrap@4.0.0-alpha.2` and `jspm i npm:twbs/bootstrap@4.0.0-alpha.2` get configured differently.